contracts/tokenization/StakedTokenFactory.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "./StakedToken.sol";
+
+contract StakedTokenFactory {
+    function createStakedToken(address stakeManagerAddress) external returns (address) {
```

Comment:
only stakemagaer should have access to this.
also add initialized

Reply:
and whennotpaused

Reply:
Yes, we can do it, 
but I am not sure if it's needed as it's going to increase the size of the factory.
even if anyone can call this anytime, they would just create a token that is not valid in our protocol.
so should we do it?

Reply:
yea not required